src/api/fromUrlStatus.ts

```diff
@@ -0,0 +1,25 @@
+import request, {prepareOptions} from './request'
+import {RequestOptions} from './request'
+import {InfoResponse} from './info'
+import {Settings} from '../types'
+
+export type FromUrlStatusResponse = InfoResponse
```

Comment:
nope, response difference, InfoResponse is info about the file, FromUrlStatusResponse is info about the upload progress, see https://api-refs.uploadcare.com/upload-api.html#operation/fromURLUploadStatus